R/render.R

```diff
@@ -329,11 +331,19 @@ use_lang_chapter <- function(chapters_list, language_code,
 
 add_links <- function(path, main_language, # nolint: cyclocomp_linter
                       language_code, site_url, type, config, output_folder,
-                      path_language) {
+                      path_language, project_dir) {
   html <- xml2::read_html(path)
 
   document_path <- path
 
+  lang_profile <- file.path(
```

Comment:
should we use `fs::path_ext_set()`? since we already use fs.

Reply:
I think I'll actually use fs::path() instead and its `ext` argument. Good point though!